llpc/include/llpc.h

```diff
@@ -232,6 +232,28 @@ class ICompiler {
   /// @returns : Result::Success if successful. Other return codes indicate failure.
   virtual Result BuildShaderModule(const ShaderModuleBuildInfo *pShaderInfo, ShaderModuleBuildOut *pShaderOut) = 0;
 
+  /// Build shader from the specified info.
```

Comment:
Can you add a comment explaining the difference between different different build functions? The current comment is not specific enough to tell without understanding the implementation.

Reply:
Update the comments. Thanks.